homeassistant/components/schlage/coordinator.py

```diff
@@ -32,10 +42,29 @@ def __init__(self, hass: HomeAssistant, username: str, api: Schlage) -> None:
     async def _async_update_data(self) -> SchlageData:
         """Fetch the latest data from the Schlage API."""
         try:
-            return await self.hass.async_add_executor_job(self._update_data)
-        except Error as ex:
+            locks: list[Lock] = await self.hass.async_add_executor_job(self.api.locks)
```

Comment:
Is the library missing a py.typed file?

Reply:
I didn't realize that was necessary. I'll add it, but I think this also doesn't need the type annotation on the `locks` variable. Removed it.